Remove shebang line from non-executable Python modules.
Upstream do not want to apply this patch because these files come from
another place and they do not want to modify them.

--- a/mercurial/lsprof.py
+++ b/mercurial/lsprof.py
@@ -1,5 +1,3 @@
-#! /usr/bin/env python
-
 import sys
 from _lsprof import Profiler, profiler_entry
 
